src/main/java/thewaypointers/trafficsimulator/JunctionLocationTestStarter.java

```diff
@@ -18,9 +13,10 @@
 
     public static void main(String[] args){
 
-        IStateProvider simulation = new JunctionTestProvider();
-        mainFrame=new MainFrame(simulation.getNextState(0));
-        TrafficSimulatorManager.setMainFrame(mainFrame);
-        TrafficSimulatorManager.setSimulation(simulation);
+        TrafficSimulatorManager trafficSimulatorManager = new TrafficSimulatorManager();
+        IStateProvider simulation = new RoadNetworkProvider();
```

Comment:
You are right, I have add the new starter for feat-48 test.
JunctionTestProvider dose have some problems. First,  when you drag the panel the car will stay at the old point while the map can move. And then, all the points in Junction list are showed as null. These all work well in RoadNetworkProvider.